src/utility/mapping.js

```diff
@@ -1,33 +1,26 @@
 import {
-  clone,
   each,
   find,
   isArray,
   isNil,
+  keys,
   map,
 } from 'lodash';
 
 const originalSuffix = '_$';
 
-/*
-There is a lot of cloning here to handle mapping.
-When implementing nesting we need to use cloneDeep.
-This method should be optimized if possible as it
-would be slow on large datasets. Test performance.
-*/
 const mapItem = (schema, item) => {
   /* eslint no-param-reassign:"off" */
   each(schema, (field) => {
-    item[`${field.name}${originalSuffix}`] = clone(item[field.name]);
+    item[`${field.name}${originalSuffix}`] = item[field.name];
```

Comment:
Clone was added because of mutations so that original is left unchanged. Will this be handled by this change?

Reply:
I can add it. It works fine with it, just couldn't find the reason why was added in the first place..

Reply:
From what I can remember, we had original and mapped properties duplicated, so original are being deleted but references should be broken before deleting props.

https://github.com/chmjs/chameleon-vuetify/issues/45
https://github.com/chmjs/chameleon-sdk/pull/24

Reply:
Yes, that's the thing I tested and it seems fine now even without clone. What am i missing 🤔